docs/routing/patterns.markdown

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
all of the route pattern items need a code exmaple IMO - it's really difficult for the reader to translate the prose to code him-/herself